packages/store-sync/src/common.ts

```diff
@@ -26,6 +26,14 @@ export type TableWithRecords = Table & { records: TableRecord[] };
 export type StoreEventsLog = Log<bigint, number, false, StoreEventsAbiItem, true, StoreEventsAbi>;
 export type BlockLogs = { blockNumber: StoreEventsLog["blockNumber"]; logs: StoreEventsLog[] };
 
+// TODO: should we add address here?
```

Comment:
I think it's best if we keep `address` outside of this to align with `eth_getLogs` filtering capability (where address is separate from topics): https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_getlogs